pkg/syncer/spec/spec_process_test.go

```diff
@@ -559,7 +559,7 @@ func TestSyncerProcess(t *testing.T) {
 				),
 			},
 		},
-		"SpecSyncer with AdvancedScheduling, sync downstream deployment": {
+		"SpecSyncer sync downstream deployment, upstream gets patched with the finalizer and the object is created downstream": {
```

Comment:
Could we move this test case before the first one, and harmonize the start of test case name for both upsert test cases ?

> SpecSyncer upsert

or

> SpecSyncer sync to downstream

?

Reply:
done